### ISSUE TITLE
Indicator and dial has lost it's green and grey colours

### ISSUE THREAD

**Issue by c11umw** [KartoffelToby/better-thermostat-ui-card#93]
As per the subject, I noticed the dial indicator lost it's normal grey colour when off and the Eco green is also missing from the indicator + the dial when active. All other colours seem to be fine.

I made a GIF so you could see what I mean.
![20230207174429_OBS_Stream](https://user-images.githubusercontent.com/48920056/217326957-7f1e3689-497c-4303-8a76-a237fdd26aef.gif)

I use card_mod to remove the background, but removing that from the card makes no difference.
It only happened recently .... since 2023.01.1 I think.

```
type: custom:better-thermostat-ui-card
entity: climate.ha_better_thermostat
disable_window: false
eco_temperature: 18.1
card_mod:
  style: |
    ha-card {
      box-shadow: none;
      background: rgba(0,0,0,0);
    }
```

**Comment by isibizi** [comment]
same here

**Comment by dabohda** [comment]
Same here with Home Assistant 2023.2.3

**Comment by ovizii** [comment]
Just happened to me too but only to one of the 3 cards. 

![chrome_cwpHbUMcVY](https://user-images.githubusercontent.com/837902/218965941-17f686f5-dd12-4a5c-9050-3c7b678dff0b.png)

**Comment by hayvan96** [comment]
Same here, on 2023.3.6

**Comment by KartoffelToby** [comment]
Fixed with 2.1.0
